src/MUSI6106Exec/CMakeLists.txt

```diff
@@ -9,8 +10,8 @@ file(GLOB ${APP}_HEADERS RELATIVE ${CMAKE_SOURCE_DIR}/src/${APP} *.h;../inc/*.h)
 add_inc_directories ()
 
 ## create the target
-add_executable(${APP} ${${APP}_SOURCES} ${${APP}_HEADERS})
+add_executable(${APP} ${${APP}_SOURCES} ${${APP}_HEADERS} tests.h)
```

Comment:
Is this change necessary? Ideally `tests.h` should get picked up already by the globbing stuff above.

Reply:
Hmm I don't think I made this change manually, I guess CLion added this when I added the file to the project directory, but good catch!